app/main.py

```diff
@@ -1,2 +1,57 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.capacity = 8
+        self.threshold = int(self.capacity * 2 / 3)
+        self.hash_table = self.create_hash_table()
+
+    def create_hash_table(self) -> list:
+        return [[] for _ in range(self.capacity)]
+
+    def resize_capacity(self) -> None:
+        this_hash_table = self.hash_table
+        self.capacity *= 2
+        self.hash_table = self.create_hash_table()
+        self.length = 0
+        self.threshold = int(self.capacity * 2 / 3)
+
+        for item in this_hash_table:
+            if item:
```

Comment:
```suggestion
            if len(item) > 0:
```
> Explicit is better than implicit.

https://peps.python.org/pep-0020/